tutorials/11_diagnostics_simulation_based_calibration.ipynb

```diff
@@ -184,34 +184,7 @@
    "cell_type": "code",
    "execution_count": 6,
    "metadata": {},
-   "outputs": [
```

Comment:
That just some clean up ?

Reply:
yes, there was user-specific meta data from a previous PR that I removed here.